astroquery/query.py

```diff
@@ -227,6 +227,7 @@ def _request(self, method, url, params=None, data=None, headers=None,
                 # Windows doesn't allow special characters in filenames like
                 # ":" so replace them with an underscore
                 local_filename = local_filename.replace(':', '_')
+            # should this default to savedir over cache_location?
```

Comment:
I added this note as a warning to us; we might want to raise it as an Issue. I think defaulting to `cache_location` is likely the wrong choice from a user's perspective.